numpy/tests/test_numpy.py

```diff
@@ -616,7 +612,7 @@ def test_array_getitem():
                                       [15, 16, 17, 18, 19]],
                                      [[20, 21, 22, 23, 24],
                                       [25, 26, 27, 28, 29]]])
-    assert arr1[3:2:-1] == np.array([])
+    assert arr1[3:2:-1].shape() == np.array([]).reshape([0, 2, 5]).shape()
```

Comment:
Made this workaround to pass the test as of now.
Will figure out how to equate empty ndarrays and fix it in a further PR.

Reply:
Merging !